src/app/helptext/network/interfaces/interfaces-form.ts

```diff
@@ -60,4 +56,65 @@ alias_netmaskbit6_tooltip : T('Select the prefix length used on the network.'),
 delete_placeholder6: T('Delete'),
 delete_tooltip6: T('Set to delete this alias.'),
 
+bridge_members_placeholder: T('Bridge Members'),
+bridge_members_tooltip: T(''),
+
+failover_critical_placeholder: T('Failover Critical'),
+failover_critical_tooltip: T(''),
+
+failover_group_placeholder: T('Failover Group'),
+failover_group_tooltip: T(''),
+
+failover_vhid_placeholder: T('Failover VHID'),
+failover_vhid_tooltip: T(''),
```

Comment:
From TrueNAS Guide "Failover" section:

"Virtual Host ID for the failover interface". Must be unique on the broadcast segment of the network. Can be any unused number between <i>1</i> and <i>255</i>."

Reply:
@wblock @william-gr this seems like a reasonable tooltip label to me, what do you say?

Reply:
yes